app/views/nsm/review_and_adjusts/show.html.erb

```diff
@@ -46,6 +46,40 @@
         <%= render partial: "nsm/#{scope}/index", locals: { claim:, records:, summary:, pagy: } %>
       </div>
     </div>
+
+    <%
+        type_changed_fields =
+          core_cost_summary.table_fields.select do |field|
+            field.dig(:name, :text).include?(t("nsm.work_items.type_changes.asterisk"))
+          end
+    %>
+    <% if type_changed_fields.any? %>
+      <section>
+        <p id="fn*">
+          <%= t("nsm.work_items.type_changes.asterisk") %>
+          <%= t("nsm.work_items.type_changes.explanation") %>
+          <%= t("nsm.work_items.type_changes.see") %>
+          <% type_changed_records.each_with_index do |work_item, index| %>
```

Comment:
My understanding is that these footnotes _should_ still appear inside the tab - it's only the `[*]` footnote that's moving outside. Because the more specific footnotes area genuinely only relevant to the work items tab.

Reply:
Worth asking Mary then as I thought the oppposite 😛

Reply:
I completely misread the code, ignore me.